src/Components/Organisms/Footer/Footer.tsx

```diff
@@ -0,0 +1,104 @@
+import { Container, Grid, Link, Stack } from '@mui/material';
+
+const Footer = () => {
+  const mediaXM = {
+    '@media screen and (max-width: 600px)': {
+      fontSize: '14px',
+    },
+  };
+  return (
+    <Container
+      maxWidth={false}
+      className="footer"
+      sx={{ backgroundColor: 'primary.main' }}
+    >
+      <Stack
+        flexDirection="row"
+        height="276px"
```

Comment:
Maybe we should use rem values for consistency ?

Reply:
Chenged to rem.